administration/logging.md

```diff
@@ -71,15 +75,18 @@ The **what** is defined by `package.subpackage` and is in most cases a binding (
 
 The **detail** of logging is defined by one of the following levels:
 
-1. DEFAULT
-2. OFF
-3. ERROR
-4. WARN
-5. INFO
-6. DEBUG
-7. TRACE
-
-The levels build a hierarchy with **ERROR** logging critical messages only and **DEBUG** logging nearly everything. **DEBUG** combineds all logs from levels 3 to 6, while **TRACE** adds further messages in addition to what **DEBUG** displays.
+| Log Level           | Log Weight    | When it should be used                                                                  |
+|---------------------|---------------|-----------------------------------------------------------------------------------------|
+| OFF                 | 0             | When no events will be logged                                                           |
+| FATAL               | 100           | When a severe error will prevent the application from continuing                        |
+| ERROR               | 200           | When an error in the application, possibly recoverable                                  |
+| WARN                | 300           | When an event that might possible lead to an error                                      |
+| INFO                | 400           | When an event for informational purposes                                                |
+| DEBUG               | 500           | When a general debugging event required                                                 |
+| TRACE               | 600           | When a fine grained debug message, typically capturing the flow through the application |
+| ALL                 | MAX           | When all events should be logged                                                        |
+
+The levels build a hierarchy with **ERROR** logging critical messages only and **DEBUG** logging nearly everything. **DEBUG** combines all logs from weight 100 to 500, while **TRACE** adds further messages in addition to what **DEBUG** displays.  **ALL** includes every log level from weight 100 to 600.
```

Comment:
Please split up to one line per Sentence.

Reply:
Done 👍